google/cloud/forseti/services/server.py

```diff
@@ -479,12 +485,39 @@ def main():
         '--enable_console_log',
         action='store_true',
         help='Print log to console.')
+
     args = vars(parser.parse_args())
 
+    if not args['services']:
```

Comment:
nit: I think moving these to their own _argument_validator() function can keep main() cleaner.